README.md

```diff
@@ -24,43 +24,43 @@ running on Cloud TPUs and learn how to use Cloud TPUs as PyTorch devices:
 
 The rest of this README covers:
 
-* [Running PyTorch on Cloud TPUs in production on Google Cloud.](#Cloud)
+* [Running PyTorch on Cloud TPUs in production on Google Cloud.](#-running-pytorch-on-cloud-tpus-with-google-cloud-platform)
 Google Cloud also runs networks faster than Google Colab.
-* [Available images and wheels](#Resource)
-* [API & Best Practices](#API)
-* [Performance Profiling and Auto-Metrics Analysis](#PerfMetrics)
-* [Troubleshooting](#Troubleshooting)
-* [Providing Feedback](#Feedback)
-* [Building and Contributing to PyTorch/XLA](#Contributing)
+* [Available images and wheels](#-available-images-and-wheels)
+* [API & Best Practices](#-api--best-practices)
```

Comment:
NB: Why two dashes after api?

Reply:
I'm guessing it has to due with the `&` in the header

Reply:
yea I think so, I need to remove `-` on the front through.. will do that soon.